src/app/resources/resources.component.ts

```diff
@@ -13,7 +13,7 @@ export class ResourcesComponent implements OnInit {
     this.topics = TOPICS;
   }
 
-  ngOnInit() {
+  public ngOnInit(): void {
```

Comment:
Yay! Thanks for adding the scope visibility! It's my preference to have them too!